cluster/src/dunit/scala/io/snappydata/cluster/ClusterManagerTestBase.scala

```diff
@@ -75,9 +76,32 @@ class ClusterManagerTestBase(s: String) extends DistributedTestBase(s) {
   // this can be used only by jobs running on Lead node
   def sc: SparkContext = SnappyContext.globalSparkContext
 
+  val startNode = new SerializableRunnable() {
+    override def run(): Unit = {
+      val node = ServiceManager.currentFabricServiceInstance
+      if (node == null || node.status != FabricService.State.RUNNING) {
+        startSnappyServer(locPort, bootProps)
+      }
+      assert(ServiceManager.currentFabricServiceInstance.status ==
+          FabricService.State.RUNNING)
+
+      val logger = LoggerFactory.getLogger(getClass)
+      logger.info("\n\n\n  STARTING TESTS IN " + getClass.getName + "\n\n")
+    }
+  }
+
+  val stopNode = new SerializableRunnable() {
+    override def run(): Unit = {
+      val node = ServiceManager.currentFabricServiceInstance
+      if (node != null || node.status == FabricService.State.RUNNING) {
+        ServiceManager.currentFabricServiceInstance.stop(null)
```

Comment:
I can see stopAny() in the ClusterManagerTestBase class. Can't those be used instead of duplicating.